constructorio-client/src/main/java/io/constructor/client/ConstructorIO.java

```diff
@@ -2251,4 +2275,125 @@ public String taskAsJson(TaskRequest req) throws ConstructorException {
             throw new ConstructorException(exception);
         }
     }
+
+    /**
+     * Creates a Quiz OkHttp request
+     *
+     * @param req the Quiz request
+     * @param type the type of quiz request (next/finalize)
+     * @return a Task OkHttp request
+     * @throws ConstructorException
+     */
+    protected Request createQuizRequest(QuizRequest req, String type, UserInfo userInfo) throws ConstructorException {
+        try {
+            if (!type.equals("next") && !type.equals("finalize"))
+                throw new IllegalArgumentException("type must be either 'next' or 'finalize'");
+
+            List<String> paths = Arrays.asList("v1", "quizzes", req.getId(), type);
+            HttpUrl url = this.makeUrl(paths);
+
+            if (req.getSection() != null) {
+                url = url.newBuilder()
+                        .addQueryParameter("section", req.getSection())
+                        .build();
+            }
+
+            if (req.getVersionId() != null) {
+                url = url.newBuilder()
+                        .addQueryParameter("version_id", req.getVersionId())
+                        .build();
+            }
+
+            if (req.getA().size() > 0) {
+                for (List<String> questionAnswers : req.getA())
+                {
+                    String answerParam = String.join(",", questionAnswers);
+                    url = url.newBuilder().addQueryParameter("a", answerParam).build();
+                }
+            } else {
+                if (type.equals("finalize")) {
+                    throw new IllegalArgumentException("a (answers) is a required parameter for a finalize request");
+                }
+            }
+
+            Request request = this.makeUserRequestBuilder(userInfo)
+                    .url(url)
+                    .get()
+                    .build();
+
+            return request;
+        } catch (Exception exception) {
+            throw new ConstructorException(exception);
+        }
+    }
+
+    /**
+     * Queries the quiz service for the next quiz/question
+     *
+     * @param req the Next Quiz request
+     * @return a Next Quiz Response
+     * @throws ConstructorException if the request is invalid.
+     */
+    public NextQuizResponse nextQuiz(QuizRequest req, UserInfo userInfo) throws ConstructorException {
```

Comment:
Should we change this from `nextQuiz` to one of the following? 🤔 
* `nextQuestion`
* `getNextQuestion`
* `nextQuizQuestion`
* `getNextQuizQuestion`